<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>623</width>
      <height>266</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="m1Label">
          <property name="text">
           <string>M1:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="kp1Label">
              <property name="text">
               <string>KP:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="kd1Label">
              <property name="text">
               <string>KD:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ki1Label">
              <property name="text">
               <string>KI:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pacl1Label">
              <property name="text">
               <string>pA_CL:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pbcl1Label">
              <property name="text">
               <string>pB_CL:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pccl1Label">
              <property name="text">
               <string>pC_CL:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QDoubleSpinBox" name="m1kpSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m1kdSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m1kiSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m1paclSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m1pbclSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m1pcclSpinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="m2Label">
          <property name="text">
           <string>M2:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="kp2Label">
              <property name="text">
               <string>KP:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="kd2Label">
              <property name="text">
               <string>KD:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ki2Label">
              <property name="text">
               <string>KI:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pacl2Label">
              <property name="text">
               <string>pA_CL:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pbcl2Label">
              <property name="text">
               <string>pB_CL:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pccl2Label">
              <property name="text">
               <string>pC_CL:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QDoubleSpinBox" name="m2kpSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m2kdSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m2kiSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m2paclSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m2pbclSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m2pcclSpinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_4">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="kp3Label">
              <property name="text">
               <string>KP:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="kd3Label">
              <property name="text">
               <string>KD:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ki3Label">
              <property name="text">
               <string>KI:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pacl3Label">
              <property name="text">
               <string>pA_CL:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pbcl3Label">
              <property name="text">
               <string>pB_CL:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pccl3Label">
              <property name="text">
               <string>pC_CL:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QDoubleSpinBox" name="m3kpSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m3kdSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m3kiSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m3paclSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m3pbclSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m3pcclSpinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="m3Label">
          <property name="text">
           <string>M3:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_5">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="m4Label">
          <property name="text">
           <string>M4:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="kp4Label">
              <property name="text">
               <string>KP:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="kd4Label">
              <property name="text">
               <string>KD:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ki4Label">
              <property name="text">
               <string>KI:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pacl4Label">
              <property name="text">
               <string>pA_CL:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pbcl4Label">
              <property name="text">
               <string>pB_CL:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pccl4Label">
              <property name="text">
               <string>pC_CL:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="QDoubleSpinBox" name="m4kpSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m4kdSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m4kiSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m4paclSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m4pbclSpinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m4pcclSpinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>278</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="versionLabel">
       <property name="styleSheet">
        <string notr="true">font: 700 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="softLabel">
              <property name="text">
               <string>Soft:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="parLabel">
              <property name="text">
               <string>Par:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="softValLabel">
              <property name="text">
               <string>00.00.00</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="spacing">
               <number>1</number>
              </property>
              <item>
               <widget class="QSpinBox" name="parTopSpinBox">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="parMidSpinBox">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="parBotSpinBox">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
